minerl/Malmo/Schemas/MissionHandlers.xsd

```diff
@@ -1255,6 +1255,17 @@
         </xs:complexType>
     </xs:element>
 
+    <xs:element name="ServerQuitFromDeath">
+        <xs:annotation>
+            <xs:documentation>
+                Specifies a time limit that applies to all agents.
+            </xs:documentation>
+        </xs:annotation>
+        <xs:complexType>
+            <xs:attribute name="quitWhenAnyDead" type="xs:boolean" use="optional" default="false"/>
```

Comment:
Note the default behavior here is  different then the current default behavior for experiments (they are more simmilar to quitWhenAnyDead=true)